rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs

```diff
@@ -56,6 +61,13 @@ impl EthRpcClient {
                 EvmRpcClient::builder_for_ic(TRACE_HTTP)
                     .with_providers(providers)
                     .with_evm_canister_id(evm_rpc_id)
+                    .with_min_attached_cycles(300_000_000_000)
```

Comment:
How about introducing a constant for `300_000_000_000`?

Reply:
Sounds good. Done.